codelist-api.rst

```diff
@@ -164,5 +164,5 @@ If you do this you may not need to alter the way you deal with codelists, but th
 * Version information is no longer provided in the XML.
 * Specific changes to codelists:
 
-  * :doc:`/codelists/BudgetIdentifier/` has different categories, and no 'sector' elements.
-  * Names in the :doc:`/codelists/FileFormat/` list are no longer there as they make little sense, and the list now tracks the IANA source it is derived from directly.
+  * The `BudgetIdentifier <https://github.com/orgs/IATI/teams>`__codelist has different categories, and no 'sector' elements.
```

Comment:
I don’t think this link is correct https://github.com/orgs/IATI/teams

Reply:
Also, I think there’s a missing space here:

> __codelist

Reply:
Thanks Andy - I've corrected these